File: lesson4/python3/homework/homework4_4.py
#!/usr/bin/python3

numb = int(input("Введтие натуральное число => "))
numb_u = 0

for numb_i in range(1, numb+1):
    numb_u += numb_i
print("Сумма натурального числа =>", numb_u)

File: lesson4/c_c++/homework/homework4_4.c
#include <stdio.h>

int main() {
    int numb, numb_u = 0;
    printf("Введите натуральное число => ");
    scanf("%d", &numb);

    for (int numb_i = 1; numb_i <= numb; numb_i++) {
        numb_u += numb_i;
    }
    printf("Сумма натурального чиса => %d\n", numb_u);
    return 0;
}
